_usage/mvu.md

```diff
@@ -0,0 +1,161 @@
+
+ *Note* that the target Babelfish database should be empty and should not already have the extensions installed. You should skip the `Enabling extensions in the target database` step while following the `INSTALLING.md` instructions for the target Babelfish installation.
+
+2. Stop the source and target servers (if running), with the following `pg_ctl` commands:
+
+- To stop the source server:
+
+`${BABELFISH_HOME_OLD}/bin/pg_ctl -D ${BABELFISH_DATA_OLD}/ stop`
+
+- To stop the target server:
+
+`${BABELFISH_HOME_NEW}/bin/pg_ctl -D ${BABELFISH_DATA_NEW}/ stop`
+
+3. Perform a major version upgrade to upgrade the Postgres server with the `pg_upgrade` utility. No special handling is needed to use pg_upgrade with Babelfish:
+
+```sql
+sudo su - postgres
+mkdir ~/upgrade
+cd ~/upgrade
+${BABELFISH_HOME_NEW}/bin/pg_upgrade -U postgres -b ${BABELFISH_HOME_OLD}/bin \
+                                       -B ${BABELFISH_HOME_NEW}/bin \
+                                       -d ${BABELFISH_DATA_OLD} \
+                                       -D ${BABELFISH_DATA_NEW} \
+                                       -p 5432 -P 5433 -j $JOBS --verbose
+```
+
+Refer to the [pg_upgrade documentation](https://www.postgresql.org/docs/14/pgupgrade.html) for detailed information about using `pg_upgrade`. 
+
+In the event of a failure, the new `data` directory (`$BABELFISH_DATA_NEW`) might have issues, but the old `data` directory (`$BABELFISH_DATA_OLD`) will remain intact. To retry the upgrade, you can remove the new `data` directory  and re-initialize it again. Follow the instructions in `Initiating the Data directory` from `INSTALLING.md`.
+
+After the upgrade is successfully completed, start the new server with `pg_ctl`:
+
+`${BABELFISH_HOME_NEW}/bin/pg_ctl -D ${BABELFISH_DATA_NEW} -l logfile start`
+
+To update the Babelfish extensions, connect to the PostgreSQL endpoint of the Babelfish database and execute the following statements from a single session in this order:
+
+```sql
+ALTER EXTENSION babelfishpg_common UPDATE;
+ALTER EXTENSION babelfishpg_tsql UPDATE;
+ALTER EXTENSION babelfishpg_tds UPDATE;
+```
+
+## Preparing the Upgraded Server for Use
+
+After upgrading, you should update the parameter settings to ensure they match the settings you noted before the upgrade. You can set the parameter settings at the `psql` command line; connect to psql with the command: 
+
+`$PKGDIR_NEW/bin/psql -U your_user_name -d babelfish_db_name` 
+
+Please note that you cannot change migration mode during an upgrade.  For example, if `babelfishpg_tsql.migration_mode` was `multi-db` before the upgrade, and you change it to `single-db`, then you will not able to see any user-defined databases that were there before the upgrade.
+
+Then use the following SQL command to change the parameter settings to match the values before the upgrade:
```

Comment:
There have been significant changes this step in the internal doc (step 10):
1. Now we are providing two ways to change the settings.
2. There is one important note as follows:
```
*Imporatant:* It is mandatory to set babelfishpg_tsql.database_name GUC using any of the methods given below even if its value is already the same as source cluster.
```

Reply:
This is now in the section titled:  ## Preparing the Upgraded Server for Use

Please be sure you're reviewing in the correct version of the file (instead of an older/cached version of the markdown).